### ISSUE TITLE
A virus in this repo?

### ISSUE THREAD

**Issue by EzraLopez** [electron/spectron#93]
My antivirus says this file:

> spectron/bin/launcher.exe

is a: 

> Trojan.Generic.17449111

**Comment by kevinsawicki** [comment]
> Trojan.Generic.17449111

Are there any more details you can provide about this virus from your virus scanner?

**Comment by EzraLopez** [comment]
Not really. I'm using Bitdefender Antivirus for Mac if that helps.

**Comment by MarshallOfSound** [comment]
@kevinsawicki Is there no way that launcher can be written in JS.

I don't really know GO but that appears to just spawn a process and then send the output to OS stdout and stderr?

**Comment by MarshallOfSound** [comment]
@mars-lander gave a slightly larger list of detected things

> Here is list of Trojan and Riskwares in this file:
"spectron/bin/launcher.exe"

> Trojan.Win32.Generic!BT
W32.Clodb96.Trojan.3010
Trojan.Ebowla
W32/Trojan.CNQO-8612
Trojan.Ebowla.d
Riskware ( 0040eff71 )
Riskware ( 0040eff71 )
Trj/CI.A
Trojan.Win32.Z.Ebowla.1744384[h]

**Comment by kevinsawicki** [comment]
>  Is there no way that launcher can be written in JS.

It is on Mac and Linux, but on Windows there were issues since the `launcher.js` is run by ChromeDriver and not Spectron and `!#` lines aren't supported on Windows.

So I wrote a really simple launcher in Go that mirrored the behavior of `launcher.js` but it is only used on Windows.

**Comment by haimich** [comment]
I have the same issue with Kaspersky on Mac.

**Comment by Crecket** [comment]
Everything still works fine for me but it does trigger warnings using Kaspersky. [Screenshot of the warning.](https://i.gyazo.com/8e7ec9d4d0b6803216510e0f9dea1f5e.png)

**Comment by yuya-oc** [comment]
Hi, is a batch file applicable for this? Unfortunately I'm not sure whether the file can avoid your antivirus and be applicable for all cases. But it can be used in `npm test` and my app's tests instead of `launcher.exe`.

```bat
@echo off
"%SPECTRON_NODE_PATH%" "%SPECTRON_LAUNCHER_PATH%" %*
if ERRORLEVEL 1 exit /b 1
exit /b 0
```

**Comment by jkleinsc** [comment]
Resolved via #197
